stable/mysql/.helmignore

```diff
@@ -1 +1,2 @@
 .git
+OWNERS
```

Comment:
I don't believe this is necessary

Reply:
Well. Documentation recommends it: https://github.com/helm/charts/blob/master/README.md#owning-and-maintaining-a-chart

Reply:
Interesting! Learning a bit everyday.